### DISCUSSION TITLE
How To Copy Colors From An Image And Apply Them On Another Image

### DISCUSSION THREAD

**Discussion by your-living-crown** [photopea/photopea#6949] [Q&A]
Hi

basically I want to do with photopea what is explained there for photoshop: https://digitalsynopsis.com/design/copy-color-and-tone-of-an-image-in-photoshop/

**Comment by photopea** [comment]
To find the three "main" colors, use Edit - Reduce Colors, enter 3. Then, save these colors manually into Swatches (pick them with the Eyedropper tool, press Window - Swatches, triangle menu - Define new).
